gtsfm/bundle/bundle_adjustment.py

```diff
@@ -96,7 +96,7 @@ def run(
             logger.error(
                 "Bundle adjustment aborting, optimization cannot be performed without any tracks or any cameras."
             )
-            return initial_data, GtsfmMetricsGroup()
+            return initial_data
```

Comment:
thanks for the PR @ayushbaid. Don't we need to return two things though?

Reply:
We do, but we did not have consensus on what to return. `GtsfmMetricsGroup` has args in the constructor which are compulsory. Either way this will break when we encounter this error.

I did not want this change to go in with my shared intrinsics PR.

Reply:
Thanks @ayushbaid. I think we should just return `None` if we don't know what to return for the second arg. But currently this change will not pass `mypy`